model.py

```diff
@@ -425,7 +425,7 @@ def generator(self, z):
             hs[0] = tf.nn.relu(self.g_bns[0](hs[0], self.is_training))
 
             i = 1 # Iteration number.
-            depth_mul = 8  # Depth decreases as spatial component increases.
+            depth_mul = self.image_size // 16  # Depth decreases as spatial component increases.
```

Comment:
61c129b22981028a2d18c590e208d0bbe192c4c9 effectively changed this from 4 to 8 for 64x64 images, which causes a tensor size mismatch.

I think that what I have written is the correct calculation to change it back to 4 for 64x64 images in a way which generalises to different image sizes, but would ideally like @md5wasp to confirm that this works for him.